README.md

```diff
@@ -4,9 +4,9 @@
 
 | Docker Tag | Factorio Version | Description | Release Date |
 | --- | :---: | --- | :---: |
-| [latest, stable](https://github.com/goofball222/factorio/blob/main/stable/Dockerfile) | [1.1.91](https://forums.factorio.com/108687) | Factorio headless server stable release | 2023-09-21 |
-| [experimental](https://github.com/goofball222/factorio/blob/main/experimental/Dockerfile) | [1.1.94](https://forums.factorio.com/109267)| Factorio headless server experimental release | 2023-10-19 |
-| [1.1.91](https://github.com/goofball222/factorio/releases/tag/1.1.91) | [1.1.91](https://forums.factorio.com/108687) | Factorio headless server stable static release | 2023-09-21 |
+| [latest, stable](./stable/Dockerfile) | [1.1.91](https://forums.factorio.com/109267) | Factorio headless server stable release | 2023-10-30 |
```

Comment:
Path here is incorrect. This change removes the full github URL reference to the Dockerfile, which should be maintained for ease of access/review from Docker Hub & github pages.